product-catalog/app/productcatalog/controllers/ProductCatalogController.java

```diff
@@ -0,0 +1,37 @@
+package productcatalog.controllers;
+
+import common.cms.CmsService;
+import common.controllers.SunriseController;
+import common.templates.TemplateService;
+import io.sphere.sdk.categories.CategoryTree;
+import io.sphere.sdk.client.PlayJavaSphereClient;
+import play.Configuration;
+import play.libs.F;
+import play.mvc.Result;
+import productcatalog.models.ProductOverviewPageContent;
+import productcatalog.templates.ProductCatalogView;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+
+@Singleton
+public class ProductCatalogController extends SunriseController {
```

Comment:
This name is so far simply a dummy placeholder for experimenting with the CMS/Pages/Templates (so, views side). Initially it was only *the* controller of the ProductCatalog module, that's where the name comes from. I wouldn't worry that much about the controller side, since I'm using it as a playground. In the next task I will start seriously with the POP and therefore with the full stack controllers/models/views :P